prime-router/src/test/kotlin/ElementTests.kt

```diff
@@ -219,4 +219,99 @@ internal class ElementTests {
         assertEquals(parent.documentation, child.documentation, "Documentation value didn't carry over from parent")
         assertEquals(parent.type, child.type, "Element type didn't carry over from parent")
     }
+
+    @Test
+    fun `test normalize and formatted round-trips`() {
+        val postal = Element(
```

Comment:
A new set of tests to try formatting and normalize roundtrip.

Reply:
Excellent!